tests/nested/manual/connections/task.yaml

```diff
@@ -0,0 +1,56 @@
+summary: Check that connections from gadget are automatically connected
+
+systems: [ubuntu-20.04-64]
+
+environment:
+  NESTED_SIGN_SNAPS_FAKESTORE: true
+  NESTED_FAKESTORE_BLOB_DIR: $(pwd)/fake-store-blobdir
+  NESTED_UBUNTU_IMAGE_SNAPPY_FORCE_SAS_URL: http://localhost:11028
+  NESTED_CUSTOM_AUTO_IMPORT_ASSERTION: $TESTSLIB/assertions/developer1-auto-import.assert
+  NESTED_CUSTOM_MODEL: $TESTSLIB/assertions/nested-20-amd64-connections.model
+  NESTED_ENABLE_TPM: false
+  NESTED_ENABLE_SECURE_BOOT: false
+
+prepare: |
+  #shellcheck source=tests/lib/nested.sh
+  . "$TESTSLIB/nested.sh"
```

Comment:
I think it is not needed because you are using the tests.nested tool instead

Reply:
Done